src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -1671,11 +1781,25 @@ def _histogramRangeMoving(self, vmin, vmax):
             with self._colormapChange:
                 colormap.setVMax(vmax)
             self._maxValue.setValue(vmax)
+        if gammaPos is not None:
+            vmin, vmax = self._histoWidget.getFiniteRange()
+            vrange = vmax - vmin
+            if vrange > 0:
+                g = (gammaPos - vmin) / (vmax - vmin)
+            else:
+                g = 0.5
+            gamma = numpy.exp(g - 0.5)
```

Comment:
Same as above, see PR https://github.com/vallsv/silx/pull/34 for an improved implementation.